core/templates/dev/head/components/SharingLinksDirective.js

```diff
@@ -45,6 +52,42 @@ oppia.directive('sharingLinks', [function() {
 
         $scope.escapedTwitterText = (
           oppiaHtmlEscaper.unescapedStrToEscapedStr($scope.getTwitterText()));
+
+        $scope.isNotDashboard = function() {
```

Comment:
FYI, I went ahead and merged #2172, so you should be able to do a new PR on top of it.

Again, please don't hesitate to let me know if you have any questions about the approach!